File: src/main/java/me/ry00001/jbot/commands/Restart.java
package me.ry00001.jbot.commands;

import me.ry00001.jbot.core.*;
import me.ry00001.jbot.Jbot;
import java.util.ArrayList;

public class Restart extends Command {
    public Restart(Jbot jbot) {
        this.name = "restart";
        this.description = "restarts jbot";
        this.ownerOnly = true;
    }

    public void run(CommandContext ctx, ArrayList<String> args) {
        ctx.send("Restarting");
        ctx.bot.shutdown();
    }
}

File: src/main/kotlin/me/ry00001/jbot/commands/Restart.kt
package me.ry00001.jbot.commands

import me.ry00001.jbot.Jbot
import me.ry00001.jbot.core.Command
import me.ry00001.jbot.core.CommandContext
import java.util.*

class Restart(jbot: Jbot) : Command("restart", "restarts jbot") {
    init {
        isOwnerOnly = true
    }

    override fun run(ctx: CommandContext, args: ArrayList<String>) {
        ctx.send("Restarting")
        ctx.bot.shutdown()
    }
}